mage_ai/server/server.py

```diff
@@ -122,7 +126,7 @@ def replace_base_path(base_path: str) -> None:
         base_path (str): The base path to replace the placeholder with.
     """
     src = os.path.join(os.path.dirname(__file__), BASE_PATH_TEMPLATE_EXPORTS_FOLDER)
-    dst = os.path.join(os.path.dirname(__file__), BASE_PATH_EXPORTS_FOLDER)
+    dst = os.path.join(get_variables_dir(), BASE_PATH_EXPORTS_FOLDER)
```

Comment:
This is a corner case but I want to make sure we catch this.
The variables_dir could be a s3 path.